.ci/test.sh

```diff
@@ -51,10 +51,13 @@ if [[ $TRAVIS == "true" ]] && [[ $TASK == "check-docs" ]]; then
 fi
 
 if [[ $TASK == "lint" ]]; then
+    sudo apt-get update
     conda install -q -y -n $CONDA_ENV \
         pycodestyle \
-        pydocstyle \
-        r-lintr
+        pydocstyle
+    conda install -q -y -n $CONDA_ENV -c conda-forge \
+        r-lintr==2.0.0
```

Comment:
Please comment https://github.com/microsoft/LightGBM/pull/2533/files#r340198019.

Reply:
sorry, resolved that thread too early! I just commented there. I think installing from CRAN is going to be a better option (testing that now), so it will effectively be `>=2.0.0` anyway.

Reply:
see https://github.com/microsoft/LightGBM/pull/2533#discussion_r342883220, I have this working now and am using `>=`